sdks/python/apache_beam/transforms/display_test.py

```diff
@@ -0,0 +1,119 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Unit tests for the DisplayData API."""
+
+from __future__ import absolute_import
+
+from datetime import datetime
+import unittest
+
+import apache_beam as beam
+from apache_beam.transforms.display import HasDisplayData, DisplayData, DisplayDataItem
+
+
+class DisplayDataTest(unittest.TestCase):
+
+  def test_inheritance_ptransform(self):
+    class MyTransform(beam.PTransform):
+      pass
+
+    display_pt = MyTransform()
+    # PTransform inherits from HasDisplayData.
+    self.assertTrue(isinstance(display_pt, HasDisplayData))
+    self.assertEqual(display_pt.display_data(), {})
+
+  def test_inheritance_dofn(self):
+    class MyDoFn(beam.DoFn):
+      pass
+
+    display_dofn = MyDoFn()
+    self.assertTrue(isinstance(display_dofn, HasDisplayData))
+    self.assertEqual(display_dofn.display_data(), {})
+
+  def test_base_cases(self):
+    """ Tests basic display data cases (key:value, key:dict)
+    It does not test subcomponent inclusion
+    """
+    class MyDoFn(beam.DoFn):
+      def __init__(self, *args, **kwargs):
```

Comment:
__init__(self, display_data=None):
    ...

Also, I'd name this "my_dd" as it's only that part of the display data.